desktop.css

```diff
@@ -693,3 +694,97 @@ form {
 .error .field.error input {
   border-color: red;
 }
+
+#popup {
+  padding-top: 7px;
+  padding-left: 2%;
+  background: #fffdfc;
+  display: block;
+  max-width: 100vw;
+  top: 50%;
+  left: 50%;
```

Comment:
- [ ] Kindly make sure the pop has a width closer to the one on the Figma design to make them look identical,
- [ ] I think it would be better to make the overlay blur as shown on the Figma design below :point_down: 

Your design:
![Screenshot from 2023-02-12 00-34-49](https://user-images.githubusercontent.com/97834160/218284022-b1fee60e-8049-44dc-b29c-0cb118bac1dc.png)

Figma:

![Screenshot from 2023-02-12 00-40-11](https://user-images.githubusercontent.com/97834160/218284038-883c49aa-d6cc-4c67-b557-42fa89aa7e19.png)